### PR TITLE
Add 4.15.3 relese note

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Add 4.15.3 release note

### CODE REVIEW

**Review by hangc0276 — APPROVED** [review]

File: site3/website/src/pages/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+* Fix ByteBuf memory leak problem when setExplicitLac. [PR #3557](https://github.com/apache/bookkeeper/pull/3577)
+* Flush time started moved to after lock. [PR #3570](https://github.com/apache/bookkeeper/pull/3570)
+* Skip replicasCheck when replication disabled. [PR #3563](https://github.com/apache/bookkeeper/pull/3563)
+* Skipping placementPolicyCheck when ledger replication disabled. [PR #3561](https://github.com/apache/bookkeeper/pull/3561)
+* Fix the deadlock when only using io thread to handle request. [PR #3480](https://github.com/apache/bookkeeper/pull/3480)
+* Fix memory leak when reading entry but the connection disconnected. [PR #3528](https://github.com/apache/bookkeeper/pull/3528)
+* Fix byteBuf potential memory leak problem. [PR #3525](https://github.com/apache/bookkeeper/pull/3525)
+* LedgerOpenOp: Do not call blocking close() in the callback. [PR #3513](https://github.com/apache/bookkeeper/pull/3513)
+* Rename success with writableResult and update final writableResult about wait writeSet. [PR 3505](https://github.com/apache/bookkeeper/pull/3505)
+* Fix close ledgerAuditorManager repeatedly. [PR #3503](https://github.com/apache/bookkeeper/pull/3503)
```
Comment (wenbingshen) [inline site3/website/src/pages/release-notes.md:38]: Uniform style?  [PR #3505]

**Review by wenbingshen — CHANGES_REQUESTED** [review]

File: site3/website/src/pages/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+## 4.15.3
+
+Release 4.15.3 includes multiple bug fixes and some dependencies CVE fixes.
+
+Apache BookKeeper users are encouraged to upgrade to 4.15.3.
+The technical details of this release are summarized below.
+
+### Notice
+
+* We introduced a break change for the RocksDB configuration in 4.15.0 but didn't consider the compatibility.
```
Comment (nicoloboschi) [inline site3/website/src/pages/release-notes.md:13]:
```suggestion
* 4.15.0 introduced a breaking change about the RocksDB configuration in 4.15.0. Release 4.15.3 allows users to safely upgrade from 4.14.x without loosing the RocksDB runtime configuration
```

File: site3/website/src/pages/release-notes.md [hunk 3]
```diff
@@ -1,6 +1,50 @@
...
+  For more details, refer to https://lists.apache.org/thread/drh4p5prxbcs8gszhxnd1xsv0g48vvbt
+  See [PR #3523](https://github.com/apache/bookkeeper/pull/3523)
+
+* We handled Timeout Exception in Recovery. The exception will change from `LedgerRecoveryException` to `TimeoutException`.
```
Comment (nicoloboschi) [inline site3/website/src/pages/release-notes.md:17]:
```suggestion
* Timeout exceptions are now handled in a better way during reads.
```

File: site3/website/src/pages/release-notes.md (hunk 1 above)
Comment (nicoloboschi) [reply site3/website/src/pages/release-notes.md:38]: I agree but it's not about style, it's about content and it must be crystal clear to users

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

File: site3/website/src/pages/release-notes.md [hunk 4]
```diff
@@ -1,6 +1,50 @@
...
+
+* Fix ByteBuf memory leak problem when setExplicitLac. [PR #3557](https://github.com/apache/bookkeeper/pull/3577)
+* Flush time started moved to after lock. [PR #3570](https://github.com/apache/bookkeeper/pull/3570)
+* Skip replicasCheck when replication disabled. [PR #3563](https://github.com/apache/bookkeeper/pull/3563)
+* Skipping placementPolicyCheck when ledger replication disabled. [PR #3561](https://github.com/apache/bookkeeper/pull/3561)
+* Fix the deadlock when only using io thread to handle request. [PR #3480](https://github.com/apache/bookkeeper/pull/3480)
+* Fix memory leak when reading entry but the connection disconnected. [PR #3528](https://github.com/apache/bookkeeper/pull/3528)
+* Fix byteBuf potential memory leak problem. [PR #3525](https://github.com/apache/bookkeeper/pull/3525)
+* LedgerOpenOp: Do not call blocking close() in the callback. [PR #3513](https://github.com/apache/bookkeeper/pull/3513)
+* Rename success with writableResult and update final writableResult about wait writeSet. [PR 3505](https://github.com/apache/bookkeeper/pull/3505)
```
Comment (zymap) [inline site3/website/src/pages/release-notes.md:37]:
```suggestion
* Rename success with writableResult and update final writableResult about wait writeSet. [PR #3505](https://github.com/apache/bookkeeper/pull/3505)
```

File: site3/website/src/pages/release-notes.md [hunk 5]
```diff
@@ -1,6 +1,50 @@
...
+* Fix ByteBuf memory leak problem when setExplicitLac. [PR #3557](https://github.com/apache/bookkeeper/pull/3577)
+* Flush time started moved to after lock. [PR #3570](https://github.com/apache/bookkeeper/pull/3570)
+* Skip replicasCheck when replication disabled. [PR #3563](https://github.com/apache/bookkeeper/pull/3563)
+* Skipping placementPolicyCheck when ledger replication disabled. [PR #3561](https://github.com/apache/bookkeeper/pull/3561)
+* Fix the deadlock when only using io thread to handle request. [PR #3480](https://github.com/apache/bookkeeper/pull/3480)
+* Fix memory leak when reading entry but the connection disconnected. [PR #3528](https://github.com/apache/bookkeeper/pull/3528)
+* Fix byteBuf potential memory leak problem. [PR #3525](https://github.com/apache/bookkeeper/pull/3525)
+* LedgerOpenOp: Do not call blocking close() in the callback. [PR #3513](https://github.com/apache/bookkeeper/pull/3513)
+* Rename success with writableResult and update final writableResult about wait writeSet. [PR #3505](https://github.com/apache/bookkeeper/pull/3505)
+* Fix close ledgerAuditorManager repeatedly. [PR #3503](https://github.com/apache/bookkeeper/pull/3503)
```
Comment (zymap) [inline site3/website/src/pages/release-notes.md:38]:
```suggestion
* Avoid closing the ledgerAuditorManager twice in the close method. [PR #3503](https://github.com/apache/bookkeeper/pull/3503)
```

**Comment by zymap** [comment]
@wenbingshen @nicoloboschi PTAL

**Review by wenbingshen — APPROVED** [review]
Thanks.

**Review by nicoloboschi — APPROVED** [review]

File: site3/website/src/pages/release-notes.md [hunk 6]
```diff
@@ -1,6 +1,50 @@
...
+## 4.15.3
+
+Release 4.15.3 includes multiple bug fixes and some dependencies CVE fixes.
+
+Apache BookKeeper users are encouraged to upgrade to 4.15.3.
+The technical details of this release are summarized below.
+
+### Notice
+
+* 4.15.0 introduced a breaking change about the RocksDB configuration in 4.15.0. Release 4.15.3 allows users to safely upgrade from 4.14.x without loosing the RocksDB runtime configuration
```
Comment (dlg99) [inline site3/website/src/pages/release-notes.md:13]:
```suggestion
* 4.15.0 introduced a breaking change in the RocksDB configuration in 4.15.0. Release 4.15.3 allows users to safely upgrade from 4.14.x without losing the RocksDB runtime configuration
```

**Review by dlg99 — APPROVED** [review]
LGTM, see spelling nit

**Pull request merged**